### PR TITLE
Add Vertex Colouring example

### PR DESCRIPTION
**Pull request by rogermc2**
colour_vertices now ignored by local .gitignore file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,10 @@ lib
...
 .DS_Store
-.#*#
\ No newline at end of file
+<<<<<<< HEAD
+.#*#
+a_dot
+moving_triangle
+=======
+.#*#
+>>>>>>> flyx/master
```
Comment (flyx) [inline .gitignore]: This is a conflict you failed to solve after merging master into your branch. Only lines 6-8 should stay.

File: examples/Superbible/Vertex Colouring/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+obj
```
Comment (flyx) [inline examples/Superbible/Vertex Colouring/.gitignore]: `obj` is already ignored by the top level `.gitignore` and does not need to be repeated here.

File: examples/Superbible/Vertex Colouring/src/main_loop.adb [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    subtype tVec4f is GL.Types.Singles.Vector4;
+
+    Rendering_Program : GL.Objects.Programs.Program;
+    Vertex_Array      : GL.Objects.Vertex_Arrays.Vertex_Array_Object;
+
+    --  ------------------------------------------------------------------------
+
+    procedure Render_Triangle (Current_Time : Glfw.Seconds) is
+        use Math_Functions;
+        Back_Colour : GL.Types.Colors.Color := (0.5 * (1.0 + Sin (Single (Current_Time))),
```
Comment (flyx) [inline examples/Superbible/Vertex Colouring/src/main_loop.adb]: The code would be more readable if you calculated `Single (Current_Time)` only once and stored it in a variable.

File: .gitignore (hunk 1 above)
Comment (rogermc2) [inline .gitignore]: I don't really understand the Git message in this file.
I've hopefully fixed it by copying the contents of the flyx/master file.
What is the .#*# for?

File: examples/Superbible/Vertex Colouring/src/main_loop.adb (hunk 3 above)
Comment (rogermc2) [inline examples/Superbible/Vertex Colouring/src/main_loop.adb]: Agreed!
Have fixed.

**Comment by flyx** [comment]
renamed the paths to all-lowercase after merging.

**Pull request merged**